script/csharp/compile-command.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script does two things
+#   1. Redirects all output to stderr
+#   2. Supplies .dll references to the csc command
+
+# Actually compile:
+# [dotnet] [csc] -reference:[stdlib.dll] [...passed in args...]
+/usr/bin/dotnet /usr/share/dotnet/sdk/3.1.101/Roslyn/bincore/csc.dll \
+-reference:/usr/share/dotnet/sdk/3.1.101/ref/netstandard.dll "$@" 1>&2
```

Comment:
```suggestion
version="$(dotnet --list-sdks | awk '{ print $1 }')"
/usr/bin/dotnet /usr/share/dotnet/sdk/"$version"/Roslyn/bincore/csc.dll \
-reference:/usr/share/dotnet/sdk/"$version"/ref/netstandard.dll "$@" 1>&2
```
Not sure if this is a good way to do it but it does work.

Reply:
Even better, we could just do `dotnet --version`?

Reply:
Yes that does seem better... I'm not sure why I didn't try that.